cmd/interlock/run.go

```diff
@@ -163,18 +163,27 @@ func runAction(c *cli.Context) {
 		}
 	} else {
 		configPath := c.String("config")
+
 		if configPath != "" {
 			log.Debugf("loading config from: %s", configPath)
 
 			d, err := ioutil.ReadFile(configPath)
 			switch {
 			case os.IsNotExist(err):
+				log.Infof("Missing Interlock configuration: file=%s", configPath)
+				log.Info("Use the run --config option to set a custom location for the configuration file")
+				log.Info("Examples of an Interlock configuration file: url=https://github.com/ehazlett/interlock/tree/master/docs/examples")
 				log.Fatalf("config not found: %s", configPath)
 			case err == nil:
 				data = string(d)
 			default:
 				log.Fatal(err)
 			}
+		} else {
```

Comment:
I brought a towel. This got me thinking it would be very helpful to have a sane default for the location of config.toml so I updated `cli.Command`.